tweets/tests.py

```diff
@@ -7,83 +7,105 @@
+        )
         self.assertEquals(response_get.status_code, 200)
-        self.assertTemplateUsed(response_get, 'tweets/tweets_detail.html')
-        self.assertContains(response_get, data['contents'])
+        self.assertTemplateUsed(response_get, "tweets/tweets_detail.html")
+        self.assertContains(response_get, data["contents"])
 
 
 class TestTweetDeleteView(TestCase):
     def setUp(self):
-        User.objects.create_user(username='yamada', email='asaka@test.com', password='wasurenaide1108')
-        self.client.login(username='yamada', password='wasurenaide1108')
-        data = {'contents': 'ワンピースの映画早く見たい'}
-        self.client.post(reverse('tweets:create'), data)
+        User.objects.create_user(
+            username="yamada", email="asaka@test.com", password="wasurenaide1108"
+        )
+        self.client.login(username="yamada", password="wasurenaide1108")
+        data = {"contents": "ワンピースの映画早く見たい"}
+        self.client.post(reverse("tweets:create"), data)
 
     def test_success_post(self):
-        tweet = Tweet.objects.get(contents='ワンピースの映画早く見たい')
-        response_post = self.client.post(reverse('tweets:delete', kwargs={'pk': tweet.pk}))
-        self.assertRedirects(response_post, reverse('accounts:home'), status_code=302, target_status_code=200)
-        self.assertFalse(Tweet.objects.filter(contents='ワンピースの映画早く見たい').exists())
+        tweet = Tweet.objects.get(contents="ワンピースの映画早く見たい")
+        response_post = self.client.post(
+            reverse("tweets:delete", kwargs={"pk": tweet.pk})
+        )
+        self.assertRedirects(
+            response_post,
+            reverse("accounts:home"),
+            status_code=302,
+            target_status_code=200,
+        )
+        self.assertFalse(Tweet.objects.filter(contents="ワンピースの映画早く見たい").exists())
 
     def test_failure_post_with_not_exist_tweet(self):
-        response = self.client.post(reverse('tweets:delete', kwargs={'pk': 9999}))
+        response = self.client.post(reverse("tweets:delete", kwargs={"pk": 9999}))
         self.assertEquals(response.status_code, 404)
-        self.assertTrue(Tweet.objects.filter(contents='ワンピースの映画早く見たい').exists())
+        self.assertTrue(Tweet.objects.filter(contents="ワンピースの映画早く見たい").exists())
 
     def test_failure_post_with_incorrect_user(self):
-        User.objects.create_user(username='satou', email='wakou@test.com', password='wasuretene1108')
-        user = User.objects.get(username='satou')
-        Tweet.objects.create(user=user, contents='ワンピース')
-        tweet = Tweet.objects.get(contents='ワンピース')
-        response = self.client.post(reverse('tweets:delete', kwargs={'pk': tweet.pk}))
+        User.objects.create_user(
+            username="satou", email="wakou@test.com", password="wasuretene1108"
+        )
+        user = User.objects.get(username="satou")
+        Tweet.objects.create(user=user, contents="ワンピース")
+        tweet = Tweet.objects.get(contents="ワンピース")

```

Comment:
```suggestion
        user = User.objects.create_user(
            username="satou", email="wakou@test.com", password="wasuretene1108"
        )
        tweet = Tweet.objects.create(user=user, contents="ワンピース")
```
この時点でインスタン化できるので、getメソッドで取ってくる必要ないです。